packages/noports_core/pubspec.yaml

```diff
@@ -24,6 +24,12 @@ dependencies:
   socket_connector: ^1.0.11
   uuid: ^3.0.7
 
+dependency_overrides:
+  socket_connector:
+    git:
+      url: https://github.com/gkc/socket_connector.git
+      ref: socket-authenticator-option
```

Comment:
Remove once socket_connector PR has been merged and the package has been published